ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
```

Comment:
I don't think adding "Lag reduction" clarifies much, as the term has several meanings, and the default you use it as (GUI responsiveness) is not my default (network latency). I specifically put "GUI responsiveness" in to address... well... GUI responsiveness.